# Stop Selected Node

string node_name
---
bool ret